R/teal_data.R

```diff
@@ -13,52 +17,63 @@
 #'  code included in the object definitions actually produces those objects.
 #'  If `check` is true and preprocessing code is empty an error will be thrown.
 #'
-#' @return (`TealData`)
+#' @return (`TealData` or `teal_data`) object
 #'
 #' @export
 #'
 #' @examples
-#' x1 <- dataset(
-#'   "x1",
-#'   iris,
-#'   code = "x1 <- iris"
-#' )
 #'
-#' x2 <- dataset(
-#'   "x2",
-#'   mtcars,
-#'   code = "x2 <- mtcars"
+#' teal_data(
+#'   x1 = iris,
+#'   x2 = mtcars,
+#'   code = quote({
+#'     x1 <- iris
+#'     x2 <- mtcars
+#'   })
 #' )
-#'
-#' teal_data(x1, x2)
```

Comment:
#' # specify data only
#' teal_data(x1, x2)
#' 
#' # specify data and code
#' teal_data(
#'   x1 = iris,
#'   x2 = mtcars,
#'   code = quote({
#'     x1 <- iris
#'     x2 <- mtcars
#'   })
#' )
#' 
#' # evaluate code in empty object
#' teal_data() |> within(x1 <- iris) |> within(x2 <- mtcars)